python/python_fastapi_server/routers/edr.py

```diff
@@ -161,6 +159,12 @@ def init_edr_collections(adaguc_dataset_dir: str = os.environ["ADAGUC_DATASET_DI
     return edr_collections
 
 
+# The edr_collections information is cached locally for a maximum of 2 minutes
+# It will be refreshed if older than 2 minutes
+edr_cache = TTLCache(maxsize=100, ttl=120)
```

Comment:
There is little point in giving a maxsize, as the function takes no argument. So there will always be only one entry.